style.css

```diff
@@ -1 +1,110 @@
-/*  Add your styling here */
+body{
+  display: flex;
+  flex-direction: column;
+  background-color: darkred;
+}
+*{ 
+    box-sizing:border-box;
+    font-family:"Dosis", sans-serif; 
+  }
+header{
+   display: flex;
+   flex-direction: column;
+   }
+  .menu{
+    display: flex;
+    justify-content:space-between;
+  }
+  .sub-menu {
+    display: none;
+  }
+  .berger-icon {
+    width: 35px;
+    height: 5px;
+    background-color: black;
+    margin: 6px 0;
+  }
+  .slider {
+     display: flex;
+     overflow: auto;
+    }
+.slide {
+   flex-shrink: 0;
+   }
+   .main-navigation {
+     margin-right: 15px;
+   }
+   .hello-section {
+    display: flex;
+    flex-direction: column;
+   }
+   h2 {
+     text-align: center;
+   }
+   .picture-text {
+     display: flex;
+     flex-direction: column;
+     align-items: center;
+   }
+.pictureWithText1 {
```

Comment:
When referring to a single element, use IDs rather than classes, e.g.:

```html
<div class="picture-with-text" id="picture-1">...</div>
<div class="picture-with-text" id="picture-2">...</div>
...
```

```css
#picture-1 {
  background: url(...);
}

#picture-2 {
  background: url(...);
}
```